docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -0,0 +1,43 @@
+# New Language Feature Check Integration Process [WIP]
+
+## Check Update Procedure
+
+### Consider Similar Tokens
+it's essential to analyze existing language constructs 
```

Comment:
Any developer docs should be considered "technical writing"; to this end, we should be very concise and direct. We have no need for phrasing like "It's essential". Everything is essential in good tech writing :)

Reply:
We need to make sure we have good capitalization and grammar, don't be afraid to run your stuff through ChatGPT 😂